plasmapy/classes/tests/test_Plasma.py

```diff
@@ -95,3 +95,93 @@ def test_Plasma_derived_vars():
     assert test_plasma.alfven_speed.shape == test_plasma.density.shape
     assert test_plasma.alfven_speed.unit.si == u.m / u.s
     assert np.allclose(test_plasma.alfven_speed.value, 10.92548431)
+
+
+class Test_PlasmaBlob(object):
+    def setup_method(self):
+        """initializing parameters for tests """
+        self.T_e = 5 * 11e3 * u.K
+        self.n_e = 1e23 * u.cm ** -3
+        self.Z = 2.5
+        self.particle = 'p'
+        self.blob = plasma.PlasmaBlob(T_e=self.T_e, 
+                                      n_e=self.n_e,
+                                      Z=self.Z,
+                                      particle=self.particle)
+        self.couplingVal = 10.468374460435724
+        self.thetaVal = 0.6032979246923964
+
+    def test_invalid_particle(self):
+        """
+        Checks if function raises error for invalid particle.
+        """
+        with pytest.raises(InvalidParticleError):
+            plasma.PlasmaBlob(T_e=self.T_e, 
+                              n_e=self.n_e,
+                              Z=self.Z,
+                              particle="cupcakes")
+        return
```

Comment:
Unnecessary returns unless you want to shoot for the off chance this also doubles as C or Fortran code, I guess?

Reply:
I just want it to return my calls :crying_cat_face: